ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentFlows.tsx

```diff
@@ -169,6 +178,18 @@ function DeploymentFlow() {
     return (
         <div className="pf-u-h-100 pf-u-p-md">
             <Stack hasGutter>
+                <StackItem>
+                    <Flex>
+                        <FlexItem flex={{ default: 'flex_1' }} />
+                        <FlexItem>
+                            <AdvancedFlowsFilter
+                                filters={advancedFilters}
+                                setFilters={setAdvancedFilters}
+                            />
```

Comment:
Would you describe how you envision using these filters to filter the data?
It seems this PR is mainly for getting the presentation of the filter dropdown working, but it would be nice to know how you see it fitting into the data flow.

Doesn't need to be a comment in the code (unless you think it deserves a permanent code comment even when implemented). A PR comment, or adding it to the PR description, would be fine.

Reply:
The component will keep track of these `advanced` filters we will use to filter the flows data. We also have a search input we'll use to filter by entity name. Using both of these values, we can have some sort of util function that takes these filters and the flows data, and just return the flows that match those filters. That will happen within the `DeploymentFlows` component